types/react.d.ts

```
@@ -0,0 +1,30 @@
+  export type MouseEvent<T = Element, E = NativeMouseEvent> = React.MouseEvent<T, E>;
+  export type ReactNode = React.ReactNode;
+  export type JSX = React.JSX;
+}
+
+// Ensure JSX namespace is available globally
+declare global {
+  namespace JSX {
+    interface Element extends React.ReactElement<any, any> {}
+    interface ElementClass extends React.Component<any> {}
+    interface ElementAttributesProperty {
+      props: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)